lib/spaceship/tunes/application.rb

```diff
@@ -126,6 +126,10 @@ def url
         "https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/ra/ng/app/#{self.apple_id}"
       end
 
+      def analytics
+        Spaceship::Tunes::AppAnalytics.factory({ 'adamId' => self.apple_id })
```

Comment:
the `.factory` method is only really necessary when creating an instance from a response. In this case, it can be re-written more simply as:
```ruby
Spaceship::Tunes::AppAnalytics.new(apple_id: self.apple_id)
```